src/main/java/Entities/PlaylistFactory.java

```diff
@@ -1,4 +1,7 @@
 package Entities;
 
 public class PlaylistFactory {
+    public Playlist createPlaylist(String name, User owner, boolean isPublic){
+        return new Playlist(name,owner,isPublic);
```

Comment:
Please put spaces after the commas in the constructor call

Reply:
In Java, white space does not matter. In fact, Java regards the following program as being identical to that of Figure 1.2.

`import turtles.*;public class BaseProgram extends TurtleProgram
{public void run(){}}`

Figure 1.2:
```
  1  import turtles.*;
  2  
  3  public class BaseProgram extends TurtleProgram {
  4      public void run() {
  5          // This is a simple baseline program.
  6      }
  7  }
```


Whitespaces (blank space) in your code are ignored by the compiler, which just skips over them while compiling the code.